skyportal/plot.py

```diff
@@ -1351,8 +1351,12 @@ def spectroscopy_plot(obj_id, user, spec_id=None, width=600, device="browser"):
         .all()
     )
 
-    if spec_id is not None:
-        spectra = [spec for spec in spectra if spec.id == int(spec_id)]
+    if spec_id is not None and len(spec_id) > 0:
+        spec_id = spec_id.split(',')
```

Comment:
Do you mind explaining what is happening here and why?

Reply:
In case the user wants to plot multiple spectra, there is an option to give a comma separated list of values. I will add a code comment. 

I added this when we added the string with the spectra names, in order to make the plot re-render on a spectrum add/delete. Then we turned the input parameter into a dummy input but I think this functionality could be useful in general so I left it in.

Reply:
I don't see an accompanying documentation change, though, so where is this feature advertised?

Reply:
Should I add a doc string for this function (which, incidentally, doesn't have one yet)?

Reply:
Yes, please!  These functions are ridiculously big, so we should look into refactoring them at some point.